deployment.go

```diff
@@ -72,7 +74,38 @@ func (r *marathonClient) Deployments() ([]*Deployment, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	// Allows loading of deployment steps from the Marathon v1.X API
+	// Implemented as a Quick fix for issue https://github.com/gambol99/go-marathon/issues/153
```

Comment:
nit-pick: I'd drop marking this as a quick fix. Those tend to outlive their purpose frequently. :-)